department-of-veterans-affairs/next-build
# VACMS-14600 add tugboat for PR environments

## Description
Closes [#14600](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14600)

## Testing done
tinkered with settings in `.tugboat/config.yml` and our `drupalClient.ts` until pages properly built inside a tugboat container.

## Screenshots
![Screenshot 2023-08-11 at 12 04 54 PM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/0d9fba06-9056-42ff-8709-77076730a7ac)

Also, just look down below in this PR's comments 😄 

## QA steps

Locally, the site should:
- still build as expected when pointing at a local cms endpoint using `yarn export` & `yarn dev`
- still build as expected when pointing at the content build endpoint using `yarn export` & `yarn dev`

Tugboat build should succeed without failing:
![Screenshot 2023-08-11 at 12 11 03 PM](https://github.com/department-of-veterans-affairs/next-build/assets/11279744/d5c3d703-0934-415f-b2aa-d8ebb1fac0d5)


### Definition of Done


### Is this PR blocked by another PR?
